### PR TITLE
ADEN-7842 AE3 with out-of-page and floor-adhesion/interstitial support

### PR DESCRIPTION
**Pull request by fraszczakszymon**
## Links

* http://wikia-inc.atlassian.net/browse/ADEN-7842

### CODE REVIEW

File: app/components/ads/invisible-high-impact-2.js [hunk 1]
```diff
@@ -20,29 +18,15 @@ export default Component.extend(RenderComponentMixin, {
     this._super(...arguments);
 
     this.get('ads.module').onReady(() => {
-      window.getInstantGlobal('wgAdDriverHighImpact2SlotCountries', (highImpactCountries) => {
-        if (this.isEnabled(highImpactCountries) && !this.isDestroyed) {
-          this.set('isVisible', true);
-          this.get('ads.module').pushSlotToQueue(this.name);
-        }
-      });
+      this.get('ads.module').pushSlotToQueue(this.name);
```
Comment (jacekbj) [inline app/components/ads/invisible-high-impact-2.js]: So much nicer 👍

File: app/mixins/ads.js [hunk 2]
```diff
@@ -54,19 +54,22 @@ export default Mixin.create({
...
-        this.ads.pushAdSlotComponent(
-          adsData.invisibleHighImpact2,
-          this.renderAdComponent({
-            name: 'ads/invisible-high-impact-2',
-            attrs: {},
-            element: placeholder,
-          }),
-        );
-      }
+      const attributes = this.get('ads.module').getAdSlotComponentAttributes(adsData.invisibleHighImpact2);
```
Comment (jacekbj) [inline app/mixins/ads.js:63]: Why do we sometimes use `this.get('ads.module')` and sometimes `this.ads` service?

File: app/modules/ads/targeting.js [hunk 3]
```diff
@@ -194,10 +194,6 @@ export default {
       targeting.cid = cid;
     }
 
-    // TODO Implement Krux integration
```
Comment (jacekbj) [inline app/modules/ads/targeting.js:197]: -0.03 kB!

**Review by jacekbj — APPROVED** [review]

**Comment by emadry** [comment]
Tested successfully - ready to merge

**Pull request merged**